lib/client.js

```diff
@@ -110,10 +116,22 @@ Client.prototype.dial = function (opts, cb) {
 
 Client.prototype._generateHeaders = function () {
   return {
-    'User-Agent': 'OpenTok-Node-SDK/' + pkg.version +
-      (this.c.uaAddendum ? ' ' + this.c.uaAddendum : ''),
-    'X-TB-PARTNER-AUTH': this.c.apiKey + ':' + this.c.apiSecret
+    'User-Agent': 'OpenTok-Node-SDK/' + pkg.version + (this.c.uaAddendum ? ' ' + this.c.uaAddendum : ''),
+    'X-OPENTOK-AUTH': this._generateJwt(),
+    Accept: 'application/json'
   };
 };
 
+Client.prototype._generateJwt = function () {
+  var currentTime = Math.floor(new Date() / 1000);
+  var token = jwt.sign({
+    iss: this.c.apiKey,
+    ist: 'project',
+    iat: currentTime,
+    exp: currentTime + defaultConfig.auth.expire
```

Comment:
https://github.com/opentok/opentok-node/pull/145/files#diff-50e86626e817aed46f9458964243aaa1R164 is hardcoded but this is config